app/controllers/api/v1/on_duty_firms_controller.rb

```diff
@@ -13,13 +13,13 @@ def show
   def rota_slots_for_location
     RotaSlot.joins(:shift)
       .where(
-        shifts: { location_uid: requested_location },
```

Comment:
I'm not 100% sure, but shouldn't the `location_uid` stay here as well? The method is called `rota_slots_for_location`.